roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -8,9 +8,11 @@
 from abc import ABCMeta, abstractmethod, abstractproperty
 from importlib import import_module
 
+# add_metaclass is not available in the embedded six from module_utils in Ansible 2.2.1
+from six import add_metaclass
 # pylint import-error disabled because pylint cannot find the package
 # when installed in a virtualenv
-from ansible.module_utils.six.moves import add_metaclass, reduce  # pylint: disable=import-error, redefined-builtin
```

Comment:
It turns out the problem was just a mistake in the import... `add_metaclass` in not part of `six.moves`, but `six`.
Fixed in #3707.